assets/scss/_grid.scss

```diff
@@ -1,3 +1,7 @@
+.container {
```

Comment:
`container` have grid properties (`grid-width` / `grid-max-width`) and it lives in the `_grid` file, I think we should name it `grid-container`